assets/js/customize-preview.js

```diff
@@ -0,0 +1,66 @@
+/* global backgroundColors */
+/**
+ * Customizer enhancements for a better user experience.
+ *
+ * Contains handlers to make Theme Customizer preview reload changes asynchronously.
+ *
+ * @since 1.0.0
+ */
+
+( function() {
+
+	// Add listener for the "header_footer_background_color" control.
+	wp.customize( 'header_footer_background_color', function( value ) {
+		value.bind( function( to ) {
+
+			// Add background color to header and footer wrappers.
+			jQuery( '#site-header,#site-footer' ).css( 'background-color', to );
```

Comment:
Agreed. I compiled a list of elements a few hours ago and posted it on https://github.com/WordPress/twentytwenty/pull/351#issuecomment-533609825

I'll need to accomodate multiple properties, so it makes sense to me to put them all in a function that can then be added to the script via `wp_localize_script`, and then the same function can be used in `custom-css.php` to get the elements and apply the necessary CSS rules.

What do you think @celloexpressions?

Reply:
I pushed a commit on a separate branch on my own fork just to discuss what we want to do before I push anything here.

How does this look to you? https://github.com/aristath/twentytwenty/commit/e46dc71e530babfdfe581a334943a1d4d97f0d45

Reply:
On the other hand this is a theme, it's not a complicated framework and we're probably overthinking this.
What would be the _simplest_ solution to do?
Not the best we have in our minds, the simplest. That's what I think we should do... Otherwise this will not make it in time for the code-freeze. It doesn't need to be perfect at this point, what we want is to merge it in time so we can have another week to work-out the finer points.
It can be an array, a string, whatever as long as it works :)

Reply:
Merged the arrays implementation. Added it both in PHP and JS, everything works.